specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/ServerTrustGroups.json

```diff
@@ -240,16 +240,16 @@
         "operationId": "ServerTrustGroups_ListByInstance",
         "parameters": [
           {
-            "$ref": "#/parameters/ResourceGroupParameter"
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
           },
           {
             "$ref": "#/parameters/ManagedInstanceNameParameter"
           },
           {
-            "$ref": "#/parameters/SubscriptionIdParameter"
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
In these serverXYZ json files, we only change the common type parameters/definitions to reference and remove previous local inline parameters/definitions.